tests/v2/validation/projects/projects.go

```diff
@@ -11,29 +11,76 @@ import (
 	"github.com/rancher/shepherd/clients/rancher"
 	management "github.com/rancher/shepherd/clients/rancher/generated/management/v3"
 	"github.com/rancher/shepherd/extensions/defaults"
-	projectsApi "github.com/rancher/shepherd/extensions/kubeapi/projects"
-	rbacApi "github.com/rancher/shepherd/extensions/kubeapi/rbac"
+	"github.com/rancher/shepherd/extensions/kubeapi/namespaces"
+	"github.com/rancher/shepherd/extensions/kubeapi/projects"
+	"github.com/rancher/shepherd/extensions/kubeapi/rbac"
+	"github.com/rancher/shepherd/extensions/kubeapi/workloads/deployments"
 	"github.com/rancher/shepherd/extensions/kubeconfig"
+	"github.com/rancher/shepherd/extensions/workloads"
 	namegen "github.com/rancher/shepherd/pkg/namegenerator"
+	appv1 "k8s.io/api/apps/v1"
+	corev1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	kwait "k8s.io/apimachinery/pkg/util/wait"
 )
 
 const (
-	dummyFinalizer   = "dummy"
-	timeFormat       = "2006/01/02 15:04:05"
-	roleProjectOwner = "project-owner"
-	roleOwner        = "cluster-owner"
+	dummyFinalizer                  = "dummy"
+	timeFormat                      = "2006/01/02 15:04:05"
+	projectOwner                    = "project-owner"
+	clusterOwner                    = "cluster-owner"
+	clusterMember                   = "cluster-member"
+	systemProjectName               = "System"
+	namespaceSteveType              = "namespace"
+	resourceQuotaAnnotation         = "field.cattle.io/resourceQuota"
+	containerDefaultLimitAnnotation = "field.cattle.io/containerDefaultResourceLimit"
+	resourceQuotaStatusAnnotation   = "cattle.io/status"
+	containerName                   = "nginx"
+	imageName                       = "nginx"
 )
 
+var resourceQuotaLimit = v3.ResourceQuotaLimit{
```

Comment:
we dont seem to be using the resourceQuotaLimits in any existing tests, why do we want this?

Reply:
This is needed for upcoming tests involving resource quota